src/components/Forecast/Widget.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import Form from './Form/Form'
+import WeatherDetailsToday from './WeatherDetailsToday/WeatherDetailsToday'
+import FiveDayForecast from './FiveDayForecast/FiveDayForecast'
+
+import './Widget.scss'
+
+const myAPIKey = 'dce6d076b7a2ffe64e08efe5ff779fa3'
+
+class Widget extends React.Component {
```

Comment:
This name doesn't tell me at all what the Component does, I'd like to reiterate that it would serve you really well to use longer names 🙂